packages/ui/src/forum/queries/forum.graphql

```diff
@@ -27,6 +27,32 @@ fragment ForumCategoryFields on ForumCategory {
   }
 }
 
+fragment ArchivedForumCategoryFields on ForumCategory {
+  ...ArchivedForumBaseCategoryFields
+  forumcategoryparent {
+    ...ForumSubCategoryFields
+    status {
+      __typename
+    }
+  }
+}
+
+fragment ArchivedForumBaseCategoryFields on ForumCategory {
+  id
+  title
+  description
+  status {
+    __typename
+    ... on CategoryStatusArchived {
+      categoryArchivalStatusUpdatedEvent {
+        createdAt
+        inBlock
+        network
+      }
+    }
+  }
+}
```

Comment:
I see that those duplicates a bit with `FourmCategoryBaseFields` so maybe we could change current `ForumCategoryFields`:

*   remove `moderators` from `ForumCategoryBaseFields`
*   remove `ArchivedForumCategoryBaseFields` (it should be the same as `ForumCategoryBaseFields`)
*   add `moderators` to `ForumCategoryFields`

This way we could have the same `Fields` for `ForumCategoryFields` and `ArchivedForumCategoryFields`.

Reply:
Thanks, done.